tests/cypress/integration/wp-module-support/utils.cy.js

```diff
@@ -71,3 +77,15 @@ export const waitForNextSteps = () => {
 		.scrollIntoView()
 		.should( 'exist' );
 };
+
+export const deleteCapabilitiesTransient = () => {
+	cy.log( 'Deleting capabilities transient' );
+	cy.exec(
+		`npx wp-env run cli wp transient delete nfd_site_capabilities`,
+		{ failOnNonZeroExit: false }
+	);
+};
+
+export const wpLogin = () => {
+	cy.login( Cypress.env( 'wpUsername' ), Cypress.env( 'wpPassword' ) );
+};
```

Comment:
add a line break.

Reply:
Again, thanks for the callout. This is updated.